code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -685,12 +685,26 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/M, alien)
 	custom_metabolism = 0.2
 	overdose_threshold = REAGENTS_OVERDOSE/5
 	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL/5
-
-/datum/reagent/medicine/hyperzine/on_mob_add(mob/living/M)
-	M.reagent_move_delay_modifier -= 0.5
+	var/duration = 0
+
+/datum/reagent/medicine/hyperzine/on_mob_delete(mob/living/M)
+	var/amount = M.adjustOxyLoss(duration * 4)
+	M.adjustHalLoss(amount)
+	if(amount > 50)
+		M.KnockOut(amount * 0.1)
```

Comment:
Taking more than 50 oxyloss already knocks out you good.

Reply:
I didn't see this anywhere. Are you sure?